src/WebJobs.Script.WebHost/Controllers/KeysController.cs

```diff
@@ -59,6 +54,48 @@ public KeysController(WebScriptHostManager scriptHostManager, ISecretManager sec
             return GetKeysResult(keys);
         }
 
+        [HttpGet]
+        [Route("admin/functions/{functionName}/keys/{name}")]
```

Comment:
nit: {keyName} here and throughout to make consistent with function level methods